config/jobs/image-pushing/k8s-staging-csi-gen.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash -e
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+output="$(dirname $0)/k8s-staging-csi.yaml"
+repos="
+csi-driver-host-path
+csi-test
+external-attacher
+external-provisioner
+external-resizer
+external-snapshotter
+livenessprobe
+node-driver-registrar
+"
+
+cat >"$output" <<EOF
+# Automatically generated by k8s-staging-csi-gen.sh.
+
+postsubmits:
+EOF
+
+for repo in $repos; do
+    cat >>"$output" <<EOF
+  kubernetes-csi/$repo:
+    - name: post-$repo-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-storage-csi-$repo
+      decorate: true
+      branches:
+        # For publishing canary images. Publishing canary images for release branches can
```

Comment:
Does it hurt to enable it on branches that don't have the build files? It will cause the job to fail, there is not much activity happening in release branches.

Reply:
Besides failed jobs there shouldn't be any negative effect. If you aren't concerned about that, then I can enable also `release-.*` branches.